lib/phlex/csv.rb

```diff
@@ -15,17 +18,21 @@ def initialize(collection)
 	attr_reader :collection
 
 	def call(buffer = +"", view_context: nil)
+		unless prevent_csv_injection? == true || prevent_csv_injection? == false
+			raise "You must define `prevent_csv_injection?` on #{self.class.inspect}, returning `true` or `false`. See https://owasp.org/www-community/attacks/CSV_Injection"
+		end
```

Comment:
By settings `prevent_csv_injection?` to `nil` by default, we can force the developer to make an informed decision upfront by raising here. The developer must choose whether to enable or disable the protection, and they have an opportunity to learn about the risk on owasp.org.